README.md

```diff
@@ -7,9 +7,3 @@ Big Peach is the PyATL meetup management system
 
 `.env` file due to: https://django-environ.readthedocs.io/en/latest/
 
-
```

Comment:
Changes that would be deleted from this pull are improper. This pull needs to reshape itself based on what is already here rather than changing everything.